cdk/package.json

```diff
@@ -12,14 +12,14 @@
     "synth": "cdk synth --path-metadata false --version-reporting false"
   },
   "dependencies": {
-    "@guardian/cdk": "49.5.0",
-    "@types/jest": "^26.0.10",
-    "@types/node": "10.17.27",
-    "aws-cdk": "^2.68.0",
-    "aws-cdk-lib": "^2.68.0",
-    "constructs": "^10.1.273",
-    "jest": "^27.1.0",
-    "ts-jest": "^27.0.5",
+    "@guardian/cdk": "51.0.0",
+    "@types/jest": "^29.5.6",
+    "@types/node": "20.8.7",
+    "aws-cdk": "2.100.0",
+    "aws-cdk-lib": "2.100.0",
+    "constructs": "^10.3.0",
+    "jest": "^29.5.0",
+    "ts-jest": "^29.1.0",
```

Comment:
Is there a reason you chose to use the caret for `jest` related dependencies, and pin the others? Could be a good idea to pin them all, as per [recommendations](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript).

Reply:
Good point! I've made them all exact versions now.